lib/legacy_mempool.go

```diff
@@ -2422,8 +2422,22 @@ func (mp *DeSoMempool) GetMempoolSummaryStats() (_summaryStatsMap map[string]*Su
 	return convertMempoolTxsToSummaryStats(mp.readOnlyUniversalTransactionList)
 }
 
-func (mp *DeSoMempool) GetFeeEstimator() *PoSFeeEstimator {
-	return nil
+func EstimateFee(txn *MsgDeSoTxn, minFeeRateNanosPerKB uint64) uint64 {
```

Comment:
What's a more self descriptive name for this? Here's one option:
```suggestion
func EstimateMaxTxnFeeV1(txn *MsgDeSoTxn, minFeeRateNanosPerKB uint64) uint64 {
```